File: 1048.py
x = float(input())
if x <= 400.00:
    s = x * 1.15
    r = s - x
    p = 15
if 400.01 <= x <= 800.00:
    s = x * 1.12
    r = s - x
    p = 12
if 800.01 <= x <= 1200.00:
    s = x * 1.10
    r = s - x
    p = 10
if 1200.01 <= x <= 2000.00:
    s = x * 1.07
    r = s - x
    p = 7
if  x > 2000.00:
    s = x * 1.04
    r = s - x
    p = 4

    
print('Novo salario: {:.2f}'.format(s))
print('Reajuste ganho: {:.2f}'.format(r))
print('Em percentual: {} %'.format(p))

File: CSharp/1048.cs
using System;
using System.Globalization;


class Program
{
    static void Main(string[] args)
    {
        CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
        decimal salario = decimal.Parse(Console.ReadLine());
        int percent=0;

        if(salario <=400)
        {percent =15;}
        else if(salario>400.01m && salario <=800.00m)
        {percent =12;}
        else if(salario >=800.01m && salario<=1200.00m)
        {percent = 10;}
        else if(salario>=1200.01m && salario<=2000.00m)
        {percent = 7;}
        else
        {
            percent=4;
        }

        Console.WriteLine($"Novo salario: {salario*(1+percent/100m):F2}");
        Console.WriteLine($"Reajuste ganho: {salario*(percent/100m):F2}");
        Console.WriteLine($"Em percentual: {percent} %");
        
    }
}